test/sequential/test-crypto-timing-safe-equal-benchmarks.js

```diff
@@ -0,0 +1,123 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+
+if (!common.hasCrypto) {
+  common.skip('missing crypto');
+  return;
+}
+
+if (!common.enoughTestMem) {
+  common.skip('skipping memory-intensive test');
```

Comment:
Based on a search in the `test/` folder it seems like all the other tests actually do this correctly, so I fixed this one to do it correctly as well.